2.5/README.md

```diff
@@ -2,11 +2,15 @@ Ruby 2.5 container image
 =================
 
 This container image includes Ruby 2.5 as a [S2I](https://github.com/openshift/source-to-image) base image for your Ruby 2.5 applications.
-Users can choose between RHEL and CentOS based builder images.
+Users can choose between RHEL, CentOS and Fedora based builder images.
 The RHEL image is available in the [Red Hat Container Catalog](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/ruby-25-rhel7)
 as registry.access.redhat.com/rhscl/ruby-25-rhel7.
 The CentOS image is then available on [Docker Hub](https://hub.docker.com/r/centos/ruby-25-centos7/)
 as centos/ruby-25-centos7.
+The Fedora image can be built with:
```

Comment:
It will be available at `registry.fedoraproject.org/...`.

Reply:
No need for user to build it themselves.

Reply:
Agree, it is same as for CentOS and RHEL images.